pkgs/development/libraries/mesa/gbm.nix

```diff
@@ -15,9 +15,19 @@
 let
   common = import ./common.nix { inherit lib fetchFromGitLab; };
 in
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
```

Comment:
Also, `finalAttrs`?

Reply:
This is getting reverted on staging in like 30 seconds.